django/dkg/cancer/ris.py

```diff
@@ -14,7 +17,42 @@ def read_article(lines):
     return article
 
 def read_ris_lines(lines):
+    '''
+    RIS file import
+    INPUT:
+    lines   lines of RIS file
+    OUTPUT:
+    pandas dataframe, columns corresponding to RIS tags
+    '''
+
     import pandas as pd
     startArticle = [idx for idx,l in enumerate(lines) if re.match(RISSTART, l)]
     articles = [read_article(lines[startArticle[s]:startArticle[s+1]]) for s in range(len(startArticle)-1)]
     return pd.DataFrame(articles)
+
+def write_article(article):
+    '''
```

Comment:
schau mal hier https://github.com/tdhd/dssg2017/blob/master/django/dkg/cancer/views.py#L94, das ist der adapter von HTML gui nach json api.

wenn du dort die keys aenderst, musst du auch in die json api schauen und in die model API, also insbesondere den feature encoder: https://github.com/tdhd/dssg2017/blob/master/django/dkg/cancer/model_api/model.py#L10, da der ja auch auch `title` und `abstract` schaut, was dann idealerweise nur noch N1 und T1 oder so heissen wuerde.

ich hab heute etwas später wieder zeit, daran zu arbeiten.